README.md

```diff
@@ -75,9 +69,9 @@ executable binary  # x86_64, ARM, CUDA, TPU
 ## Documentation
 
 - [Build and Test](CONTRIBUTING.md)
-
-- Examples
-
-  - [MNIST trainining](./mnist_test.go)
-
 - [Design Docs](./doc/design.md)
+- How To
+  - [Wrap PyTorch Native Functions](./doc/wrap_native_functions.md)
+- Examples
+  - [MNIST Training](./mnist_test.go)
```

Comment:
Do we need to move the MNIST example to the `example` folder? If so, I can do it after this PR merged.

Reply:
I agree. We need it to be a standalone program for demonstrations on Raspberry Pi. Thanks!